dashboard/app/views/home/_homepage.html.haml

```diff
@@ -42,3 +42,20 @@
       = link_to pd_workshop_admins_path do
         %h3
           Directory for Workshop Admins
+    - courses_facilitated = Pd::CourseFacilitator.where(facilitator: current_user).map(&:course)
+    -if courses_facilitated.any?
+      %h1
+        Facilitator Landing Page
+
+      -if courses_facilitated.include? Pd::Workshop::COURSE_CSF
+        %a{href: "https://curriculum.code.org/plcsf/", target: '_blank'}
+          %h3
```

Comment:
I guess we use heading styles to make these links look big, and it's certainly an existing pattern, but these are usually not really headings, so I think it would be better if we could make a new CSS class and use that for all of these.

Reply:
I'm going to leave the styling as is for now and let the PLC team decide how you would like this area to be styled going forward.